open-metadata-resources/open-metadata-labs/lineage/lineage.ipynb

```diff
@@ -0,0 +1,692 @@
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e1cd0151-292c-4d32-b141-be61529946e4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "deployServerToPlatform(adminPlatformURL, \"garygeeke\", \"open-lineage\", corePlatformURL)\n",
+    "activateServerOnPlatform(\"open-lineage\", \"unused\", corePlatformURL)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "68b2c05d-0adc-4524-9144-278a5b829aa1",
+   "metadata": {},
+   "source": [
+    "## Create some sample lineage\n",
+    "\n",
+    "Now that we have the lineage services configured and running, we will create some example metadata that can be seen and used within the user interface.\n",
+    "\n",
+    "### Register an external tool\n",
+    "\n",
+    "We'll begin by registering an external data platform that was used to create the data files and processing that we will see in the lineage. Peter Profile will carry out this step against cocoMDS1 in the Data Lake platform."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "435ac5f6-9e75-4c6a-95fb-4ea672b12c9f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def registerTool(serverName, serverPlatformURL, userId, body):\n",
+    "    registerURL = serverPlatformURL + '/servers/' + serverName + '/open-metadata/access-services/data-engine/users/' + userId\n",
+    "    registerURL += '/registration'\n",
+    "    response = issuePost(registerURL, body)\n",
+    "    print(\"Response: \", response.json())\n",
+    "    guid = response.json().get('guid')\n",
+    "    if guid:\n",
+    "        return guid\n",
+    "    else:\n",
+    "        print (\"No data engine created\")\n",
+    "        processErrorResponse(serverName, serverName, serverPlatformURL, response)\n",
+    "\n",
+    "engineDoc = \"\"\"\n",
+    "{\n",
+    "    \"dataEngine\":\n",
+    "    {\n",
+    "        \"qualifiedName\": \"(organization)=Coco Pharmaceuticals::(project)=ExternalDataPlatform\",\n",
+    "        \"displayName\": \"ExternalDataEngine\",\n",
+    "        \"description\": \"An external data engine capability\",\n",
+    "        \"engineType\": \"DataEngine\",\n",
+    "        \"engineVersion\": \"1\",\n",
+    "        \"enginePatchLevel\": \"2\",\n",
+    "        \"vendor\": \"Example\",\n",
+    "        \"version\": \"1\",\n",
+    "        \"source\": \"source\"\n",
+    "    }\n",
+    "}\n",
+    "\"\"\"\n",
+    "\n",
+    "registerTool(\"cocoMDS1\", dataLakePlatformURL, petersUserId, json.loads(engineDoc))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3ac395bd-87ad-438c-9fb3-bf039a1f6f32",
+   "metadata": {},
+   "source": [
+    "### Create some example files\n",
+    "\n",
+    "Next, we'll simulate that external tool sharing metadata about some of the files that it has processed: both the files themselves, as well as the fields within them.\n",
+    "\n",
+    "These will be created by Peter Profile against cocoMDS1 in the Data Lake platform."
```

Comment:
more a scenario comment, but I wonder if this would be more suited to cocoMDS5 (business systems)?